ENIGMAsystem/SHELL/Universal_System/Resources/AssetArray.h

```diff
@@ -68,8 +68,8 @@ class OffsetVector {
     return data()[index];
   }
   void resize(size_t count) {
-    data_owner_.resize(count - LEFT);
-    data_ = data_owner_.data() - LEFT;
+    data_owner_.resize(count + LEFT);
+    data_ = data_owner_.data() - LEFT * sizeof(T);

```

Comment:
When we resize the vector, are we telling it the number of right values or the number of left and right values? I've changed it here to be the former, only the size of normal/right values.